### PR TITLE
Added is_prime function to simple_functions.py

### PR DESCRIPTION
**Pull request by KathrynJones1**
Added is_prime function to simple_functions.py. If the value given to this function is not a prime number, it will return False. If it is prime, it will return True. 

Outstanding problem: I haven't checked if the input is itself an integer.

### CODE REVIEW

**Review by mfouesneau — COMMENTED** [review]
Can you use the PEP8 conventions for formatting your code?
spaces around operator signs?

**Comment by KathrynJones1** [comment]
Like this?

**Review by mfouesneau — APPROVED** [review]
thanks

File: participants/Jones/pr_tutorial/simple_functions.py [hunk 1]
```diff
@@ -10,3 +10,10 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(value):
+    for n in range(2, int(value / 2) + 1):
```
Comment (mfouesneau) [inline participants/Jones/pr_tutorial/simple_functions.py:15]: clever trick.

**Review by mfouesneau — APPROVED** [review]

**Comment by ceb8** [comment]
Thanks! 😄

**Pull request merged**